x-pack/plugins/code/public/components/file_tree/file_tree.tsx

```diff
@@ -6,13 +6,44 @@
 import React from 'react';
 
 import { EuiIcon, EuiSideNav } from '@elastic/eui';
+import classes from 'classnames';
 
 import { connect } from 'react-redux';
 import { RouteComponentProps, withRouter } from 'react-router-dom';
+import styled from 'styled-components';
 import { FileTree as Tree, FileTreeItemType } from '../../../model';
 import { closeTreePath, fetchRepoTree, FetchRepoTreePayload } from '../../actions';
 import { EuiSideNavItem, MainRouteParams, PathTypes } from '../../common/types';
 import { RootState } from '../../reducers';
+
+const FolderClosedTriangle = styled.span`
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin-right: 4px;
+  border: 6px solid transparent;
+  border-left: 6px solid grey;
+  border-right: 6px solid transparent;
+  vertical-align: middle;
+`;
+
+const FolderOpenTriangle = styled.span`
```

Comment:
`FolderOpenTriangle` seem to have a couple of shared properties with `FolderClosedTriangle`. You can use

```
const FolderOpenTriangle = styled(FolderClosedTriangle)``
```

to inherit and override some of them.